guides/Shopify.md

```diff
@@ -0,0 +1,53 @@
+# Setting deploys to Shopify via CircleCI
+
+## Problem:
+
+By the reason of specificity of work with Shopify via `grunt-shopify`, we uploading changed file each time we save it. This may cause situations, when we push some changes to the server, then discard them with turned off `grunt`. So, we have some changes on the server and no records in git about them.  
+
+## Solution:
+
+The solution is to synchronise deploy branches (`master` and `staging`) with servers and automatically deploy via CircleCI on each of the pushes to the related branch.
```

Comment:
To combat the issue, we configured the CI server, in this case CircleCI, to manage all of the deployments for us. We hooked the `master` branch to deploy direct to the production store and the `staging` branch to the staging store.